Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -510,6 +553,18 @@ AppDelegateViewModelOutputs {
 
     // Koala
 
+    self.notificationAuthorizationStatusProperty.signal
+      .skipNil()
+      .skip(until: self.notificationAuthorizationCompletedProperty.signal)
+      .take(first: 1)
+      .observeValues {
+        if $0.isAuthorized {
+          AppEnvironment.current.koala.trackPushPermissionOptIn()
+        } else if $0.isDenied {
+          AppEnvironment.current.koala.trackPushPermissionOptOut()
```

Comment:
Again, if we use the `UNAuthorizationStatus` enum then you can use a `switch/case` here and better ensure we cover all the cases.